packages/jellyfish-core/src/category/mining.ts

```diff
@@ -0,0 +1,48 @@
+import { JellyfishClient } from '../core'
+
+export interface MintingInfo {
+  blocks: number
+  currentblockweight?: number
+  currentblocktx?: number
+  difficulty: string
+  isoperator: boolean
+  masternodeid?: string
+  masternodeoperator?: string
+  masternodestate?: 'PRE_ENABLED' | 'ENABLED' | 'PRE_RESIGNED' | 'RESIGNED' | 'PRE_BANNED' | 'BANNED'
```

Comment:
Nice. I need to add this on my masternode PR (I only have enabled, pre_resigned and resigned)